### PR TITLE
Orsay feedback - second round

### PR DESCRIPTION
**Pull request by karpov-sv**
Was originally mistakenly based on previous master state, so the log contains some already merged entries (first actually new one is about cutouts removal).

The commit about different colors for negative and positive detections needs probably some feedback, I cannot decide whether I am happy with the result or not.

### CODE REVIEW

File: apps/plotting.py [hunk 1]
```diff
@@ -3372,8 +3343,8 @@ def draw_tracklet_radec(pdf) -> dict:
         return msg
 
     # shortcuts
-    ra = pdf['i:ra'].apply(lambda x: float(x))
-    dec = pdf['i:dec'].apply(lambda x: float(x))
+    ra = pdf['i:ra'].astype(int)
+    dec = pdf['i:dec'].astype(int)
```
Comment (JulienPeloton) [inline apps/plotting.py:3347]: I think ra & dec should be `float` here

File: index.py [hunk 2]
```diff
@@ -282,7 +282,7 @@
                     ],
                     options={
                         'class:':fink_classes, 'class=':fink_classes,
-                        'last:':['10', '100', '1000'], 'last=':['10', '100', '1000'],
+                        'last:':['10', '100', '1000'], 'last=':['1', '10', '100', '1000'],
```
Comment (JulienPeloton) [inline index.py:285]: `'last:':['10', '100', '1000'],` should be `'last:':['1', '10', '100', '1000'],` to mirror options for `last=`

**Review by JulienPeloton — APPROVED** [review]
Thanks for all the optimisations on the code! I left only two minor bugs I found.

**Comment by karpov-sv** [comment]
Indeed! Thanks for noticing these bugs! Fixed

**Comment by JulienPeloton** [comment]
Thanks! I'm merging and deploying

**Pull request merged**